adafruit_rfm69.py

```diff
@@ -330,6 +297,63 @@ def __init__(
         self._write_u8(_REG_TEST_PA2, _TEST_PA2_NORMAL)
         # Set the syncronization word.
         self.sync_word = sync_word
+        self.preamble_length = preamble_length  # Set the preamble length.
+        self.frequency_mhz = frequency  # Set frequency.
+        self.encryption_key = encryption_key  # Set encryption key.
+        # set radio configuration parameters
+        self._configure_radio()
+        # initialize last RSSI reading
+        self.last_rssi = 0.0
+        """The RSSI of the last received packet. Stored when the packet was received.
+           This instataneous RSSI value may not be accurate once the
+           operating mode has been changed.
+        """
+        # initialize timeouts and delays delays
+        self.ack_wait = 0.5
+        """Sets the delay time before attemting a retry after not receiving an AC"""
```

Comment:
```suggestion
        """The delay time before attempting a retry after not receiving an ACK"""
```
For attributes no need to say "Set..." it's covered by assignment and is deceiving because they can be read as well.

Reply:
Done